motor.py

```diff
@@ -50,50 +52,64 @@ def set_status(self, state: int) -> None:
 class Row:
     """Row class to define the status of its three cells"""
 
-    cell1 = Cellstatus
-    cell2 = Cellstatus
-    cell3 = Cellstatus
+    cell1 : Cellstatus
+    cell2 : Cellstatus
+    cell3 : Cellstatus
+    def __init__(self):
+        """Define a cellstatus object for each cell in a new row"""
+        self.cell1 = Cellstatus()
+        self.cell2 = Cellstatus()
+        self.cell3 = Cellstatus()
 
-    def display(self) -> None:
-        print( f"In this row: {self.cell1.get_status()} | {self.cell2.get_status()} | {self.cell3.get_status()}")
+    def __str__(self) -> str:
+        return f"{self.cell1.get_status()} | {self.cell2.get_status()} | {self.cell3.get_status()}"
 
     def set_row_status(self, colomn: int, state: int) -> None:
         """Method to change a given cell in colomn 1-3 for blank(1), circle(2) or cross(3) state"""
         match colomn:
```

Comment:
This is still a incomplete case expression, at least add a default with an exception please

Reply:
this should be properly fixed in #4  but for now the addition of default case is enough

Reply:
I think I just did it in the last commit (although I’ll check it well in the corresponding issue)